axonserver/src/test/java/io/axoniq/axonserver/grpc/SubscriptionQueryITCase.java

```diff
@@ -0,0 +1,87 @@
+package io.axoniq.axonserver.grpc;
+
+import org.axonframework.axonserver.connector.AxonServerConfiguration;
+import org.axonframework.axonserver.connector.AxonServerConnectionManager;
+import org.axonframework.axonserver.connector.query.AxonServerQueryBus;
+import org.axonframework.axonserver.connector.query.QueryPriorityCalculator;
+import org.axonframework.common.Registration;
+import org.axonframework.messaging.Message;
+import org.axonframework.messaging.responsetypes.ResponseTypes;
+import org.axonframework.queryhandling.GenericSubscriptionQueryMessage;
+import org.axonframework.queryhandling.QueryBus;
+import org.axonframework.queryhandling.QueryResponseMessage;
+import org.axonframework.queryhandling.SimpleQueryBus;
+import org.axonframework.queryhandling.SubscriptionQueryResult;
+import org.axonframework.queryhandling.SubscriptionQueryUpdateMessage;
+import org.axonframework.serialization.xml.XStreamSerializer;
+import org.junit.*;
+
+import java.lang.ref.PhantomReference;
+import java.lang.ref.Reference;
+import java.lang.ref.ReferenceQueue;
+
+import static org.junit.Assert.*;
+
+/**
+ * Integration test for Subscription Queries
+ *
+ * @author Sara Pellegrini
+ * @since 4.2
+ */
+
+public class SubscriptionQueryITCase {
```

Comment:
This test cannot be started unless we have AxonServer running, or using maven. We could use TestContainers maybe (but they require docker installed). Not sure that I have the best answer here, but it would be convenient to be able to start this test from IntelliJ.

Reply:
JUnit test with a Standalone?

Reply:
I think it's the best option!